l10n_th_personal_income_tax/models/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    pit_line = fields.One2many(
+        comodel_name="pit.move",
+        inverse_name="partner_id",
+        string="Personal Income Tax",
+        domain=[("payment_state", "!=", "draft")],
+        # readonly=False,
+    )
+    pit_yearly = fields.One2many(
```

Comment:
Do we still need pit_yearly?

I think we already have detail pit.move, which we can do a BI view for yearly. Possible?

Reply:
I think manager can see overall employee. So, I add module `pit.move.yearly` for BI view.
or It's not necessary it see overview?

Reply:
I think we already have detail pit.move, which we can do a BI view for yearly. Possible?